frontend/src/AlertsManagement/History.js

```diff
@@ -1,12 +1,34 @@
 import React, { Component } from 'react';
-import Box from '@material-ui/core/Box';
-import List from '@material-ui/core/List';
-import ListItem from '@material-ui/core/ListItem';
-import ListItemText from '@material-ui/core/ListItemText';
+import MaterialTable from 'material-table';
+import Chip from '@material-ui/core/Chip';
 import Typography from '@material-ui/core/Typography';
+import LinkIcon from '@material-ui/icons/Link';
+import { DATA_DELIMITER } from './management_constants';
 import { getAlertsData } from './management_helpers';
+import { tableIcons } from './table_icons';
 import { formatDate } from '../time_utils';
 
+const createData = (id, timestampDate, numAnomalies, status, priority, metrics, dimensions) => {
+  return { id, timestampDate, numAnomalies, status, priority, metrics, dimensions };
+}
+
+const rows = (allAlerts) => { 
+  let rowItems = [];
+  allAlerts.forEach((alert, alertId) => {
+    let metrics = new Set(alert.anomalies.map((anomaly) => anomaly.metricName));
+    let dimensions = new Set(alert.anomalies.map((anomaly) => anomaly.dimensionName));
+
+    rowItems.push(
+      createData(alertId, formatDate(alert.timestampDate, false), 
+        alert.anomalies.length, alert.status, 1,
```

Comment:
nit: make `1` a constant, or add a `TODO` since I assume it is a placeholder for priority value?

Reply:
Yeah I'll add a TODO

Reply:
thanks!

Reply:
nit: /* parameterName = */ 1